src/pg/sql/40_observatory_utility.sql

```diff
@@ -151,3 +151,25 @@ BEGIN
 
 END;
 $$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION cdb_observatory._GetRelatedColumn(columns_ids text[], reltype text )
+RETURNS TEXT[]
```

Comment:
As a more general question, is it better to return a `TABLE` with one column or an array?